common/main/cpp/native_c4collection.cc

```diff
@@ -339,11 +339,41 @@ Java_com_couchbase_lite_internal_core_impl_NativeC4Collection_createVectorIndex(
     options.vector.minTrainingSize = (unsigned) minTrainingSize;
     options.vector.maxTrainingSize = (unsigned) maxTrainingSize;
     options.vector.numProbes = (unsigned) numProbes;
+    options.vector.lazy = isLazy == JNI_TRUE;
 
     createIndex(env, coll, kC4VectorIndex, jName, kC4N1QLQuery, jqueryExpressions, options);
 #endif
 }
 
+/*
+ * Class:     com_couchbase_lite_internal_core_impl_NativeC4Collection
+ * Method:    getIndex
+ * Signature: (JLjava/lang/String;)J
+ */
+JNIEXPORT jlong JNICALL
+Java_com_couchbase_lite_internal_core_impl_NativeC4Collection_getIndex(
+        JNIEnv *env,
+        jclass ignore,
+        jlong coll,
+        jstring jName) {
+    jstringSlice name(env, jName);
+
+    C4Error error{};
+    C4Index *idx = c4coll_getIndex((C4Collection *) coll, name, &error);
+    if (idx == nullptr) {
+        // no error code; no error.  :shrug:
+        if (error.code != 0)
```

Comment:
should this be == 0?

Reply:
Boy I'm glad you caught that.  I do that kind of stuff *constantly*

Fixed.